cmd/aspect/configure/configure.go

```diff
@@ -154,12 +158,47 @@ configure:
 		}
 	}
 
+	preArgs := []string{}
+
+	// gazelle --cpuprofile enabled via environment variable.
+	cpuprofile := os.Getenv("GAZELLE_CPUPROFILE")
```

Comment:
These are all aspect-cli features that have been removed from aspect-gazelle. Basically ways to specify gazelle cli args via env.

All the code changes in this PR is just code moving, it should be functionally equivalent. But the aspect-gazelle upgrade has various fixes.